### PR TITLE
Fix typos in metrics doc

### PR DESCRIPTION
**Pull request by avlitman**
metrics doc contain names with double underscore.

Fixes #...

### Changes proposed
- names will be correct.

### Checklist

- [ ] Includes tests covering the new functionality?
- [ ] Ready for review
- [ ] Follows CONTRIBUTING rules

### Notify

@Waschndolos

### CODE REVIEW

File: docs/metrics/index.md [hunk 1]
```diff
@@ -47,26 +47,26 @@ Required Plugin:
...
+| default_jenkins_builds<buildname>_last_build_duration_milliseconds         | Build times in milliseconds of last build                                                               | gauge           |
+| default_jenkins_builds<buildname>_last_build_start_time_milliseconds       | Last build start timestamp in milliseconds                                                              | gauge           |
+| default_jenkins_builds<buildname>_last_build_tests_total                   | Number of total tests during the last build                                                             | gauge           |
+| default_jenkins_builds<buildname>_last_last_build_tests_skipped            | Number of skipped tests during the last build                                                           | gauge           |
+| default_jenkins_builds<buildname>_last_build_tests_failing                 | Number of failing tests during the last build                                                           | gauge           |
```
Comment (Waschndolos) [inline docs/metrics/index.md:65]: The Plugin outputs like `default_jenkins_builds_last_build_tests_failing` so the `_` seems right or am I missing something here?

**Review by Waschndolos — CHANGES_REQUESTED** [review]
Not sure why the `_` has been removed here. Looks right to me when I check the endpoint. Like for example:

`default_jenkins_builds_last_build_result_ordinal` or `default_jenkins_builds_last_build_result`

File: docs/metrics/index.md (hunk 1 above)
Comment (avlitman) [reply docs/metrics/index.md:65]: I see 2 underscores in the readme , see https://github.com/jenkinsci/prometheus-plugin/blob/master/docs/metrics/index.md

**Comment by avlitman** [comment]
> Not sure why the `_` has been removed here. Looks right to me when I check the endpoint. Like for example:
> 
> `default_jenkins_builds_last_build_result_ordinal` or `default_jenkins_builds_last_build_result`

the end pint is right but the doc is bad, there is 2 underscores in the readme which is wrong.

**Comment by avlitman** [comment]
@Waschndolos Oh I see you changes it here https://github.com/jenkinsci/prometheus-plugin/commit/304ce257bcb3e2864619591d774bfa4b76e7110e after I opened this pr..
so that's why we can close my pr right?

**Comment by Waschndolos** [comment]
Thanks for reaching back. I'll need to test this again - maybe I'm overseeing something here

**Comment by Waschndolos** [comment]
I'll go ahead and close this PR and fix the documentation in https://github.com/jenkinsci/prometheus-plugin/issues/666 . Guess it's too confusing right now

**Pull request closed without merging**